examples/text-generation/utils.py

```diff
@@ -163,11 +163,9 @@ def setup_model(args, model_dtype, model_kwargs, logger):
     if args.use_hpu_graphs:
         from habana_frameworks.torch.hpu import wrap_in_hpu_graph
 
-        # TODO: remove the following check from SynapseAI v1.15
         if check_habana_frameworks_version("1.13.0"):
             if model.config.model_type == "falcon":
-                args.skip_hash_with_views = True
-            model = wrap_in_hpu_graph(model, hash_with_views=not args.skip_hash_with_views)
+                model = wrap_in_hpu_graph(model, hash_with_views=False)
```

Comment:
This should be done for all models, not only Falcon no?

Reply:
no, only Falcon model had problem in 1.13.

Reply:
Yes I know, but in this piece of code, models that are not Falcon will not be wrapped in HPU graphs when using 1.13. It doesn't matter much though since we are going to switch to 1.14.

Reply:
@regisss , you are right. It is a problem we need to fix. Just by browsing the code quickly, i didn't notice the problem. @schoi-habana , please fix it asap. thanks.